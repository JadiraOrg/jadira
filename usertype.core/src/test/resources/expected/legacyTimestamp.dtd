<!ELEMENT dataset (
    LEGACYTIMESTAMP*)>

<!ELEMENT LEGACYTIMESTAMP EMPTY>
<!ATTLIST LEGACYTIMESTAMP
    ID CDATA #REQUIRED
    LEGACYTIMESTAMP CDATA #IMPLIED
    NAME CDATA #IMPLIED
>

