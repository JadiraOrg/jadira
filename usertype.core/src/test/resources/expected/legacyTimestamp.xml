<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE dataset SYSTEM "legacyTimestamp.dtd">
<dataset>
  <legacyTimestamp ID="0" LEGACYTIMESTAMP="2004-02-25 12:11:10.0" NAME="test_0"/>
  <legacyTimestamp ID="1" LEGACYTIMESTAMP="1980-03-11 13:12:11.0" NAME="test_1"/>
</dataset>